teams/tidb/membership.json

```diff
@@ -98,7 +98,7 @@
         "ChenPeng2013",
         "Defined2014",
         "dhysum",
-	"elsa0520",
+	    "elsa0520",
```

Comment:
cleanup?

Reply:
It seems be caused by using `\t` by PR762